rcl/include/rcl/time.h

```diff
@@ -46,6 +46,14 @@ typedef rcutils_time_point_value_t rcl_time_point_value_t;
 typedef rcutils_duration_value_t rcl_duration_value_t;
 
 /// Time source type, used to indicate the source of a time measurement.
+/**
+ * RCL_ROS_TIME will report the latest value reported by a ROS time source, or
```

Comment:
I'm not sure if this is the appropriate place, but should we describe what a "ROS time source" is?  I'm not sure everyone will be familiar with it.

Reply:
Maybe just link to the design document.

Reply:
I've added a link to the design doc: c20cafe